[2022-11-05 14:54:45,335] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: get_tax_receipts.find_large_receipts 2022-11-04T14:54:33.452820+00:00 [queued]>
[2022-11-05 14:54:45,379] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: get_tax_receipts.find_large_receipts 2022-11-04T14:54:33.452820+00:00 [queued]>
[2022-11-05 14:54:45,382] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2022-11-05 14:54:45,383] {taskinstance.py:1043} INFO - Starting attempt 1 of 1
[2022-11-05 14:54:45,385] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2022-11-05 14:54:45,413] {taskinstance.py:1063} INFO - Executing <Task(PythonOperator): find_large_receipts> on 2022-11-04T14:54:33.452820+00:00
[2022-11-05 14:54:45,422] {standard_task_runner.py:52} INFO - Started process 96 to run task
[2022-11-05 14:54:45,438] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'get_tax_receipts', 'find_large_receipts', '2022-11-04T14:54:33.452820+00:00', '--job-id', '39', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/texas_dags.py', '--cfg-path', '/tmp/tmp0hahphej', '--error-file', '/tmp/tmptr7yxogn']
[2022-11-05 14:54:45,440] {standard_task_runner.py:77} INFO - Job 39: Subtask find_large_receipts
[2022-11-05 14:54:45,550] {logging_mixin.py:104} INFO - Running <TaskInstance: get_tax_receipts.find_large_receipts 2022-11-04T14:54:33.452820+00:00 [running]> on host c0f23e4d7a13
[2022-11-05 14:54:45,656] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=get_tax_receipts
AIRFLOW_CTX_TASK_ID=find_large_receipts
AIRFLOW_CTX_EXECUTION_DATE=2022-11-04T14:54:33.452820+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-11-04T14:54:33.452820+00:00
[2022-11-05 14:54:45,660] {logging_mixin.py:104} INFO - found large receipts
[2022-11-05 14:54:45,662] {python.py:118} INFO - Done. Returned value was: None
[2022-11-05 14:54:45,698] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=get_tax_receipts, task_id=find_large_receipts, execution_date=20221104T145433, start_date=20221105T145445, end_date=20221105T145445
[2022-11-05 14:54:45,779] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-11-05 14:54:45,818] {local_task_job.py:146} INFO - Task exited with return code 0
